tests/clients/test_basic_client.py

```diff
@@ -72,14 +73,16 @@ def test_metrics_reporter_evaluate() -> None:
             test_current_server_round: {
                 "evaluate_start": datetime.datetime(2012, 12, 12, 12, 12, 12),
                 "loss": test_loss,
-                "evaluate_metrics": test_metrics,
+                "evaluate_metrics": test_metrics_final,
             },
         },
     }
 
 
 def test_evaluate_after_fit_enabled() -> None:
     fl_client = MockBasicClient()
+    fl_client.validate = MagicMock()  # type: ignore
+    fl_client.validate.return_value = fl_client.mock_loss, fl_client.mock_metrics
```

Comment:
Are these values actually required, since you're not asserting that they are equal to anything?

Reply:
they are unfortunate required or else this test will fail